Tests/Smoketest3.xctestplan

```diff
@@ -47,6 +47,7 @@
         "HistoryTests\/testRecentlyClosedWebsiteClosed()",
         "HistoryTests\/testRecentlyClosedWebsiteOpen()",
         "HistoryTests\/testRemoveAllTabsButtonRecentlyClosedHistory()",
+        "HistoryTests\/testTabHistory()",
```

Comment:
I don't think we can move the whole suite... now it is smoketest 4 taking almost double than other test suites
![Screenshot 2023-05-16 at 11 43 01](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/93149950-406a-433c-800f-82f42505a049)
We should pick a smaller test suite to move or split the tests...